requirements.txt

```diff
@@ -3,18 +3,16 @@ w3lib==1.13.0
 queuelib==1.4.2
 Scrapy==1.0.5
 
-# Portia
-slybot==0.12.1
-scrapely==0.12.0
-
 # Scrapinghub
 hubstorage==0.22.0
-dateparser==0.3.2
+# dateparser==0.3.4 replaced with newer commit
+https://github.com/scrapinghub/dateparser/archive/1223377.zip/#egg=dateparser
```

Comment:
Oh, have just pushed, sorry. I constantly forget that don't have write access to this repo, could you add me please?